src/transformers/models/deformable_detr/modeling_deformable_detr.py

```diff
@@ -28,22 +28,24 @@
 from torch.autograd.function import once_differentiable
 
 from ...activations import ACT2FN
-from ...file_utils import (
+from ...modeling_attn_mask_utils import _prepare_4d_attention_mask
```

Comment:
@amyeroberts Also I have noticed that file_utils are used in here which is now utils

Reply:
Yes, good catch, definitely better to import from attn mask utils.